Dockerfile

```diff
@@ -51,3 +54,14 @@ RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib && \
     setup ts_idl -t current && \
     setup ts_salobj -t current && \
     scons --clean && scons || scons
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib && \
+    source /home/saluser/repos/ts_sal/setup.env && \
+    setup ts_sal -t current && \
+    setup ts_idl -t current && \
+    setup ts_salobj -t current && \
+    package-docs build && \
```

Comment:
I no longer see my other comment, so I'll risk duplicating it by stating it again: there is no longer any need to build the SALPY_Script library because salobj doesn't use it. It just uses SALPY_Test to test salobj<->SALPY communication. That's in the make_salpy_libs.py command at about line 48 (existing code).

Reply:
The `credentials` in Jenkinsfile is how to get the credentials for the lsst.io server. They are stored in Jenkins.

It downloads a fresh container every time it starts so the environment is clean of old docs. I'll remove the SALPY build.